spec/spec_helper.rb

```diff
@@ -1,11 +0,0 @@
-RSpec.configure do |config|
-  config.expect_with :rspec do |expectations|
-    expectations.include_chain_clauses_in_custom_matcher_descriptions = true
-  end
-
-  config.mock_with :rspec do |mocks|
-    mocks.verify_partial_doubles = true
-  end
-
-  config.shared_context_metadata_behavior = :apply_to_host_groups
-end
```

Comment:
I don't think we should remove this really. We might need these settings in the future. It's something that Rails will generate by default.